### DISCUSSION TITLE
make Virtualenvs optional

### DISCUSSION THREAD

**Discussion by Saphyel** [astral-sh/rye#68] [Ideas]
I think this should be a minor change and it would be a big improvemnt for people using containers or people that doesn't know how to use Virtualenvs

**Comment by mitsuhiko** [comment]
I hope that it’s possible to make virtualenvs as obvious and clear as node_modules or the rust target folder. By adding a second non virtualenv mode the entire user experience will become more complex as now a user needs to understand both modes.

**Reply by dgqyushen** [reply]
totally agree
